Gems/Multiplayer/Code/Include/Multiplayer/Components/ISimplePlayerSpawner.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/RTTI/RTTI.h>
+
+namespace Multiplayer
+{
+    //! @class ISimplePlayerSpawner
+    //! @brief The SimplePlayerSpawnerRequest event-bus exposes helper methods regarding network player spawners.
+    //! Although the Multiplayer System automatically spawns in players, it's common for game specific server logic to retrieve valid spawn locations when respawning a player.
+    class ISimplePlayerSpawner
+    {
+    public:
+        AZ_RTTI(ISimplePlayerSpawner, "{6DAC5CDE-5D63-4C0B-9F7B-9F5B09079DBB}");
+
+        virtual ~ISimplePlayerSpawner() = default;
+
+        //! Returns the location where the next joining player will be spawned. Unlike RoundRobinNextSpawnPoint(), this will not cause the current spawn point index to increment.
```

Comment:
Nit: There is no RoundRobinNextSpawnPoint() any longer.

Reply:
Thank you!